client-src/elements/chromedash-timeline.ts

```diff
@@ -1,26 +1,27 @@
-import {LitElement, css, html, nothing} from 'lit';
-import {showToastMessage} from './utils.js';
-import {SHARED_STYLES} from '../css/shared-css.js';
+import { LitElement, css, html, nothing } from 'lit';
+import { SHARED_STYLES } from '../css/shared-css.js';
+import { showToastMessage } from './utils.js';
+import { customElement, property, state } from 'lit/decorators.js';
+interface Property {
```

Comment:
Rather than define this twice, how about putting it in a new file named something like datatypes.ts?